packages/richtext-lexical/src/features/toolbars/inline/client/Toolbar/index.tsx

```diff
@@ -95,6 +95,7 @@ function ToolbarGroupComponent({
           <ToolbarDropdown
             anchorElem={anchorElem}
             editor={editor}
+            group={group}
             groupKey={group.key}
```

Comment:
Could you remove group.key and group.items, since we're passing the entire group anyways?

Reply:
Hm you are right, good point! It would technically be a breaking change.

Could you mark this PR as a breaking change? It's a simple one and will affect only very very few users, so I think it's fine